src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -130,7 +131,7 @@ public DissolutionPatchResponse patchDissolutionApplication(
         }
 
         try {
-            return dissolutionService.addDirectorApproval(companyNumber, userId, request.getRemoteAddr(), body.getOfficerId());
+            return dissolutionService.addDirectorApproval(companyNumber, userId, body.getIpAddress(), body.getOfficerId());
```

Comment:
can we just pass body through to the service to reduce the number of params? Allow the mappers to extract the fields that they need